code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -31,40 +33,32 @@
 	adjust_stagger(max(0, ex_slowdown - 2)) //Stagger 2 less than slowdown
 
 	//Sunder
-	var/sunder_loss = 50*(3 - severity) * bomb_sunder_multiplier
-	adjust_sunder(max(0, sunder_loss))
+	adjust_sunder(max(0, 50 * (3 - severity) * bomb_sunder_multiplier))
 
 	//Damage
 	var/ex_damage = 40 + rand(0, 20) + 50*(4 - severity)  //changed so overall damage stays similar
-	ex_damage *= bomb_damage_multiplier
-
-	apply_damage(ex_damage/2, BRUTE, updating_health = TRUE)
-	apply_damage(ex_damage/2, BURN, updating_health = TRUE)
+	apply_damages(ex_damage * 0.5, ex_damage * 0.5, blocked = BOMB, updating_health = TRUE)
 
 
-/mob/living/carbon/xenomorph/apply_damage(damage = 0, damagetype = BRUTE, def_zone, blocked = 0, sharp = FALSE, edge = FALSE, updating_health = FALSE)
-	if(status_flags & (GODMODE))
+/mob/living/carbon/xenomorph/apply_damage(damage = 0, damagetype = BRUTE, def_zone, blocked = 0, sharp = FALSE, edge = FALSE, updating_health = FALSE, penetration)
+	if(status_flags & (GODMODE) ||damagetype != (BRUTE||BURN))
```

Comment:
I would personally have done that in another if since it is a complete different idea. Just a styling choice though

Reply:
done